packages/demo/package.json

```diff
@@ -27,9 +27,9 @@
         "faker": "4.1.0",
         "jquery": "3.5.1",
         "lorem-ipsum": "2.0.3",
-        "moment": "2.26.0",
-        "react": "16.12.0",
-        "react-dom": "16.12.0",
+        "moment": "2.25.3",
```

Comment:
why did it go from 2.26.0 to 2.25.3?

Reply:
I guess there is a conflict. Ill rebase on master branch